e2e/cypress/tests/integration/signin_authentication/login_close_server_spec.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @signin_authentication
+
+describe('Login page with close server', () => {
+    before(() => {
+        // Disable other auth options
+        const newSettings = {
+            Office365Settings: {Enable: false},
+            LdapSettings: {Enable: false},
+            TeamSettings: {EnableOpenServer: false},
+        };
+        cy.apiUpdateConfig(newSettings);
+
+        // # Create new team and users
+        cy.apiInitSetup().then(() => {
+            cy.apiLogout();
+            cy.visit('/login');
+        });
+    });
+    it('MM-47222 Should verify access problem is present', () => {
+        cy.findByText('Don\'t have an account?').should('exist').and('be.visible').should('have.attr', 'href', '/access_problem');
```

Comment:
(not a request for change)
In e2e tests, we try to have assertions that a regular human could do.
In this case, I would probably ask a human to see if they indeed have the 'Don't have an account?' text but I would certainly have them click on the link and make sure they end up on the page that contains "To access your team's workspace, contact your workspace admin". Doing so we validate that the link exists and as a bonus, we make sure the next page is also not broken.
Sometimes we don't have a choice and have to assert on "technical" parts, but whenever possible it's better not to in e2e tests.
Does that make sense?

Reply:
it does. I will improve this.